kernel/kernel-api/src/main/java/io/delta/kernel/internal/TableFeatures.java

```diff
@@ -124,21 +131,18 @@ public static void validateWriteSupportedTable(
         // Identity is the only one writer feature added in version 6
         throw unsupportedWriterProtocol(tablePath, minWriterVersion);
       case 7:
+        Set<String> supportedWriterFeatures =
```

Comment:
This should be a static value `SUPPORTED_WRITER_FEATURES` declared at the top of this file.